src/pages/legacy-documentation/developer-tools/beet.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Beet
-metaTitle: Developer Tools - Beet
+metaTitle: Developer Tools | Beet
```

Comment:
```suggestion
metaTitle: Beet | Developer Tools
```
I feel like in the legacy developer tools section the order should be the other way around. If you have Beet and Cusper open you do not see which one is open in which tab?